R/calcOutput.R

```diff
@@ -57,298 +58,322 @@
   argumentValues <- c(as.list(environment()), list(...))  # capture arguments for logging
- 
+
   # read region mappings check settings for aggregate
-  if (aggregate!=FALSE) {
+  if (aggregate != FALSE) {
     rel <- list()
-    rel_names <- NULL
-    for(r in c(getConfig("regionmapping"),getConfig("extramappings"))) {
-      rel[[r]] <- toolGetMapping(r, type="regional", activecalc=type)
+    relNames <- NULL
+    for (r in c(getConfig("regionmapping"), getConfig("extramappings"))) {
+      rel[[r]] <- toolGetMapping(r, type = "regional", activecalc = type)
       # rename column names from old to new convention, if necessary
-      if(any(names(rel[[r]])=="CountryCode")) names(rel[[r]])[names(rel[[r]])=="CountryCode"] <- "country"
-      if(any(names(rel[[r]])=="RegionCode")) names(rel[[r]])[names(rel[[r]])=="RegionCode"] <- "region"
-      if(is.null(rel[[r]]$global)) rel[[r]]$global <- "GLO"  # add global column
-      rel_names <- union(rel_names,names(rel[[r]]))
+      if (any(names(rel[[r]]) == "CountryCode")) names(rel[[r]])[names(rel[[r]]) == "CountryCode"] <- "country"
+      if (any(names(rel[[r]]) == "RegionCode")) names(rel[[r]])[names(rel[[r]]) == "RegionCode"] <- "region"
+      if (is.null(rel[[r]]$global)) rel[[r]]$global <- "GLO"  # add global column
+      relNames <- union(relNames, names(rel[[r]]))
     }
   }
-  
-  if(!is.logical(aggregate)) {
+
+  if (!is.logical(aggregate)) {
     # rename aggregate arguments from old to new convention, if necessary
-    if(toupper(aggregate)=="GLO") aggregate <- "global"
-    if(toupper(gsub("+","",aggregate,fixed = TRUE))=="REGGLO") aggregate <- "region+global"
-    
-    # Ignore columns in 'aggregate' that are not defined in one of the mappings. 
+    if (toupper(aggregate) == "GLO") aggregate <- "global"
+    if (toupper(gsub("+", "", aggregate, fixed = TRUE)) == "REGGLO") aggregate <- "region+global"
+
+    # Ignore columns in 'aggregate' that are not defined in one of the mappings.
     # Stop if 'aggregate' contains none of the columns defined in one of the mappings.
-    aggregate_splitted <- strsplit(aggregate,"+",fixed=TRUE)[[1]]
-    common_columns <- aggregate_splitted %in% rel_names
-    if(all(!common_columns)) {
-      stop("None of the columns given in aggregate = ",aggregate," could be found in the mappings!")
+    aggregateSplitted <- strsplit(aggregate, "+", fixed = TRUE)[[1]]
+    commonColumns <- aggregateSplitted %in% relNames
+    if (all(!commonColumns)) {
+      stop("None of the columns given in aggregate = ", aggregate, " could be found in the mappings!")
     } else {
-      if(any(!common_columns)) vcat(verbosity = 0,'Omitting ',aggregate_splitted[!common_columns],' from aggregate = ',aggregate,' because it does not exists in the mappings.')
+      if (any(!commonColumns)) vcat(verbosity = 0, "Omitting ", aggregateSplitted[!commonColumns],
+        " from aggregate = ", aggregate, " because it does not exists in the mappings.")
```

Comment:
This style is pretty confusing to me, because the continuation line is indented less than the original call. I'd prefer {} again, or a newline after `if (...)` or a newline immediately after `vcat(`. This kind of situation comes up frequently, and I'd argue similarly in all other cases as well.